presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -750,7 +752,7 @@ public void testQueryLoggingCount()
                             ImmutableList.of()),
                     new Duration(1, MINUTES));
 
-            long beforeCompletedQueriesCount = queryManager.getStats().getCompletedQueries().getTotalCount();
```

Comment:
Dear reviewer: The effect of this change is very similar to just putting 

```sleepUninterruptibly(100, MILLISECONDS)```

before this line. 
I may change it, whichever you prefer.